packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -100,5 +100,6 @@ export const messages = {
   force_apex_execute_document_text:
     'SFDX: Execute Anonymous Apex with Editor Contents',
   force_apex_execute_selection_text:
-    'SFDX: Execute Anonymous Apex with Currently Selected Text'
+    'SFDX: Execute Anonymous Apex with Currently Selected Text',
+  force_refresh_sobjects: 'SFDX: Refresh SObject Definitions'
```

Comment:
`force_sobjects_refresh`? Since that seems to follow more with how the CLI does it and how these other messages do it. `force_<category>_<action>`

Reply:
good suggestion - slightly more consistent